VimeoNetworking/Sources/AuthenticationController.swift

```diff
@@ -259,7 +259,7 @@ final public class AuthenticationController
      */
     public func accessToken(token: String, completion: @escaping AuthenticationCompletion)
     {
-        let customSessionManager =  VimeoSessionManager.defaultSessionManager(baseUrl: self.configuration.baseUrl, accessTokenProvider: {token})
+        let customSessionManager =  VimeoSessionManager.defaultSessionManager(baseUrl: self.configuration.baseUrl, accessTokenProvider: {token}, apiVersion: VimeoDefaultAPIVersionString)
```

Comment:
Is there a reason why we're opting to use `VimeoDefaultAPIVersionString ` vs `configuration.apiVersion`?